CONTRIBUTING-es.md

```diff
@@ -0,0 +1,23 @@
+# Contribuyendo
+
+Por favor, mantenga el [seguidor de problemas](http://github.com/hakimel/reveal.js/issues) limitado a **informes de errores**, **solicitudes de funciones** y **solicitudes de pull**.
+
+## Apoyo personal
+
+Si tienes soporte personal o preguntas de configuración, el mejor lugar para hacerlas es [StackOverflow](http://stackoverflow.com/questions/tagged/reveal.js).
```

Comment:
`Si Usted requiere de ...` . Support linnk should be https://t.me/CubanTech , please change in English docs as well.